mllib/src/main/scala/org/apache/spark/ml/feature/StringIndexer.scala

```diff
@@ -234,7 +234,7 @@ class StringIndexerModel (
     val metadata = NominalAttribute.defaultAttr
       .withName($(outputCol)).withValues(filteredLabels).toMetadata()
     // If we are skipping invalid records, filter them out.
-    val (filteredDataset, keepInvalid) = getHandleInvalid match {
```

Comment:
Why need change this line ?

Reply:
Thanks for picking this out! I changed this because I was matching on `$(handleInvalid)` in VectorAssembler and that seems to be the recommended way of doing this. Should I include this in the current PR and add a note or open a separate PR?

Reply:
ok. it doesn't matter no need separate PR I think. just a minor change.

Reply:
For the record, in general, I would not bother making changes like this.  The one exception I do make is IntelliJ style complaints since those can be annoying for developers.